### DISCUSSION TITLE
Ability to override home dashboard with empty value

### DISCUSSION THREAD

**Discussion by gtirloni** [grafana/grafana#73063] [General]
If a Grafana instance has a default dashboard configured (Administration > Default Preferences > Home Dashboard), users cannot *not* have a dashboard by default. The only way to override this setting is to define another home dashboard. If a user wishes to see no dashboard upon opening Grafana, that is not possible once the instance is configured in this way.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82549. Feel free to continue the discussion around this there. Thank you!
